Iterable/iterable-web-sdk
# Run Node Build on CI

Adds Node Build to CI since it should also work without problems with each PR

## To Test

1. Run `yarn && yarn build:node`
2. Ensure `index.node.js` is properly created
